main.go

```diff
@@ -97,9 +97,6 @@ func main() {
 		contentType := ""
 		for firstTime := true; firstTime || bytesRead == len(fileBytes); {
 		    bytesRead, err = file.Read(fileBytes)
-		    if err != nil {
-			fmt.Printf("%s\n", err);
-		    }
```

Comment:
I didn't catch this on the first pass but it looks like this wasn't actually trapping the error anyway, so later on when we write fileBytes into the hasher we're presumably writing either an empty buffer or whatever happened to be left in the buffer from a previous read.

It looks like the existing pattern elsewhere in this thing was to print a message to stderr and return `nil`, which I guess causes it to skip over the files it has trouble with. (I don't really remember exactly what went into that now, honestly...) So perhaps the appropriate thing here is to do similarly, skipping over this particular file with a warning and then continuing with the others.